Dockerfile.backwards

```diff
@@ -16,21 +16,8 @@ RUN	rosdep update && \
 # NOTE: This is a workaround for setuptools 50.* (see https://github.com/pypa/setuptools/issues/2352)
 ENV SETUPTOOLS_USE_DISTUTILS=stdlib
 
-RUN pip3 install --upgrade pip setuptools
+RUN pip3 install --upgrade pip==20.* setuptools==44.0.0
 RUN pip3 install -r requirements.txt
-RUN pip3 install colcon-bundle
```

Comment:
You mention in the description that this stuff is removed because of some colcon thing - can you explain that in more detail? Does colcon explicitly depend on pip>=21?

Reply:
No, it'll get colcon build and then fail because that version is getting pip 21